packages/components/organisms/f-registration/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+Latest (add to next release)
+------------------------------
+*January 12, 2021*
+
+### Changed
+- Update axios version for security advisory.
+- Use latest version of `f-button`.
```

Comment:
This version already seems to be bumped, as there's no change to this line in the package.json.